docs/content/authoring-bundles.md

```diff
@@ -3,11 +3,162 @@ title: Authoring Bundles
 description: Authoring Bundles with Porter
 ---
 
-##  TODO
+Porter generates a bundle from its manifest, porter.yaml. The manifest is made up of a few components:
 
-* Explain manifest
-* Link to VS Code extension
+* [Bundle Metadata](#bundle-metadata)
+* [Mixins](#mixins)
+* [Parameters](#parameters)
+* [Credentials](#credentials)
+* [Bundle Actions](#bundle-actions)
+* [Dependencies](#dependencies)
+* [Generated Files](#generated-files)
+
+We have full [examples](https://github.com/deislabs/porter/tree/master/examples) of Porter manifests in the Porter repository.
+
+## Bundle Metadata
+
+The metadata is defined by the [CNAB Spec](https://github.com/deislabs/cnab-spec/blob/master/101-bundle-json.md).
+
+```yaml
+name: porter-azure-wordpress
+description: Install Wordpress on Azure
+version: 0.1.0
+invocationImage: deislabs/porter-azure-wordpress:latest
+```
+
+* `name`: The name of the bundle
+* `description`: A description of the bundle
+* `version`: The version of the bundle, uses [semver](https://semver.org)
+* `invocationImage`: The name of the container image to tag the bundle with when it is built. The format is
+`REGISTRY/IMAGE:TAG`. Porter will push to this location during `porter build` so select a location that you have access to.
+
+## Mixins
+
+Mixins are adapters between the Porter and an existing tool or system. They know how to talk to Porter to include everything
+they need to run, such as a CLI or config files, and how to execute their steps in the Porter manifest.
+
+Anyone can [create a mixin](/mixin-dev-guide/), here's a list of the mixins that are installed with Porter by default:
+
+* exec - runs shell scripts and commands
```

Comment:
nit: perhaps reword as `run shell scripts...` to match verb forms for the subsequent two mixin descriptions